packages/synapse-react-client/src/components/UserCard/UserCardMedium.tsx

```diff
@@ -251,7 +243,8 @@ export function UserCardMedium({
   return (
     <Card>
       <div
-        className={`SRC-primary-background-color SRC-userCard SRC-userCardMediumUp`}
+        className={`SRC-userCard`}
+        style={{ backgroundColor: 'transparent' }}
```

Comment:
Is this background color set anywhere else? It seems like this override is unnecessary, but I might be missing something
```suggestion
```

Reply:
I thought this was necessary to remove the blue background, but that doesn't seem to be the case anymore